.github/workflows/pull_request.yml

```diff
@@ -146,6 +146,8 @@ jobs:
       run: |
         git clone https://github.com/allenai/allennlp-models.git
         cd allennlp-models
+        # TODO: remove this line.
+        git checkout conllu
```

Comment:
This line can be removed after https://github.com/allenai/allennlp-models/pull/61 is merged.

Reply:
I just merged that.